src/side-navigation/parts.tsx

```diff
@@ -79,7 +97,14 @@ interface Item {
   items?: Array<Item>;
 }
 
-export function NavigationItemsList({ items, variant, activeHref, fireChange, fireFollow }: NavigationItemsListProps) {
+export function NavigationItemsList({
+  items,
+  variant,
+  activeHref,
+  fireChange,
+  fireFollow,
+  position = '',
```

Comment:
When is this one undefined?

Reply:
Here: https://github.com/cloudscape-design/components/blob/main/src/side-navigation/implementation.tsx#L75

Reply:
Would it be better to have an explicit "root" position?

Reply:
`position` is chained: https://github.com/cloudscape-design/components/pull/2725/files#diff-08a958e7ca3d2a2f7a1bb81060fe998806ac3848e8f12c59a121a8707a47dfcaR117

Reply:
Still find it confusing, but okay